be/Cf.WebApi/Endpoints/ServiceDocumentEndpoints.cs

```diff
@@ -17,44 +19,25 @@ public static void MapServiceDocumentRoutes(this IEndpointRouteBuilder builder)
             .WithTags(Tag)
         .HasApiVersion(1);
 
-        group.MapPost("UploadPdf", UploadPdfAsync);
         group.MapPost("UploadpdfBytes", UploadPdfBytesAsync);
```

Comment:
Not your change but still could be updated.
```suggestion
        group.MapPost("UploadPdfBytes", UploadPdfBytesAsync);
```